assets/js/modules/analytics/components/setup/SetupFormGA4Transitional.js

```diff
@@ -51,6 +51,7 @@ export default function SetupFormGA4Transitional() {
 	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
 	const profileID = useSelect( ( select ) => select( STORE_NAME ).getProfileID() );
 
+	const ga4PropertyID = useSelect( ( select ) => select( MODULES_ANALYTICS_4 ).getPropertyID() );
 	const ga4ExistingTag = useSelect( ( select ) => select( MODULES_ANALYTICS_4 ).getExistingTag() );
 	const ga4MeasurementID = useSelect( ( select ) => select( MODULES_ANALYTICS_4 ).getMeasurementID() );
```

Comment:
Maybe for readability, we could put the condition here into a `shouldShowAssociatedPropertyNotice` variable, similar to how it is below in `assets/js/modules/analytics/components/setup/SetupFormUA.js`.

Reply:
👍